backend/tests/Feature/ModuleHookEventsTest.php

```
@@ -0,0 +1,127 @@
+
+        $customer->update(['name' => 'Renamed']);
+        Event::assertDispatched(ResourceChanged::class, fn ($e) => $e->model->is($customer) && $e->action === 'updated');
+
+        $customer->delete();
+        Event::assertDispatched(ResourceChanged::class, fn ($e) => $e->action === 'deleted');
+    }
+
+    public function test_work_order_scheduled_fires_on_planner_update(): void
+    {
+        $this->seed(\Database\Seeders\RolesAndPermissionsSeeder::class);
+        $admin = User::factory()->create();
```

PHPStan (2.2.6)

[error] Call to an undefined static method App\Models\User::factory().


(staticMethod.notFound)